force-app/main/default/classes/SOQL.cls

```diff
@@ -108,7 +139,12 @@ public with sharing class SOQL {
      * @return A list of Opportunity records.
      */
     public static List<Opportunity> getOpportunitiesByNameKeyword(String keyword) {
-        return null; // Replace null with the variable you used to store the result
+        String searchKeyword = '%' + keyword + '%';
+        List<Opportunity> result = [
+            SELECT Name, StageName, CloseDate, AccountId 
+            FROM Opportunity
+            WHERE Name LIKE :searchKeyword];
+        return result; // Replace null with the variable you used to store the result
```

Comment:
Formatting your vertical query like this is also fine if you have few enough fields that you are SELECTing.